insomnia_workspace.json

```diff
@@ -1333,7 +1356,7 @@
         "faucet_url": "http://127.0.0.1:8099"
       },
       "dataPropertyOrder": {
-        "&": [
+        "\u0026": [
```

Comment:
Did you change this on purpose? The environment variables are looking different now, but I did not test a call because I don't wanna setup mantis right now

Reply:
it's unicode for &, changed it back 👍